src/Symfony/Component/Clock/MockClock.php

```diff
@@ -20,13 +20,18 @@
  */
 final class MockClock implements ClockInterface
 {
-    private DatePoint $now;
+    private ?DatePoint $now = null;
+    /**
+     * @var (\Closure(\DateTimeZone $timezone): DatePoint)|null
```

Comment:
~let's forbid passing a timezone when a closure is passed, that makes things simpler:~
```suggestion
     * @var (\Closure(): DateTimeImmutable)|null
```

Reply:
arf, nvm, withTimeZone makes this a requirement

Reply:
still, what about not passing a timezone to the closure and calling setTimezone on the result instead?

Reply:
Indeed, I like it better